docs/docs/policy-reference/_examples/encoding/base64_encode_http_auth/title.txt

```diff
@@ -0,0 +1 @@
+OPA Envoy Header Manipulation
```

Comment:
I think we should rename

docs/docs/policy-reference/_examples/encoding/base64_encode_http_auth/*

to match the title here.